android/java/res/layout/brave_leo_clear_history_dialog.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. -->
+
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
We can remove LinearLayout here as it only contains a single view. we can use Textview directly.

Reply:
Sweet - I was cargo culting brave/android/java/res/layout/brave_leo_reset_dialog.xml